apps/api-documenter/src/markdown/MarkdownEmitter.ts

```diff
@@ -115,7 +115,7 @@ export class MarkdownEmitter {
         if (context.insideTable) {
           const code: string = this.getTableEscapedText(docCodeSpan.code);
           const parts: string[] = code.split(/\r?\n/g);
-          writer.write(parts.join('`<p/>`'));
+          writer.write(parts.join('</code><br/><code>'));
```

Comment:
I agree that you've found the best possible solution, but I'll point out that it has the downside that each line of code will have a separate styling rectangle. It looks okay on GitHub, but results may vary with other Markdown renders.

I've never liked the pipes-and-dashes table construct.  It has many limitations, and the syntax isn't particularly easy to write.  If I remember right, [CommonMark](https://commonmark.org/) chose not to standardize the pipes-and-dashes table after a very long discussion about it.  My general philosophy is that while Markdown saves a lot of typing for basic formatting problems, we should defer to using HTML for any complex nesting structures.  (This is the philosophy that I'm advocating for TSDoc.)  

Unfortunately, as soon as we introduce an HTML table tag, it turns off Markdown entirely:

```markdown
<table><tr><td>
`code`
**bold**
</td></tr></table>
```

gets rendered as:

<table><tr><td>
`code`
**bold**
</td></tr></table>

Long term, I'd like to add an `api-documenter html` command line that generates (style-neutral) HTML instead of Markdown.  HTML is much easier to generate, whereas Markdown was intended for an interactive human editing experience.  Markdown is very difficult for a machine to generate correctly as we see in this PR.  The main reason we chose Markdown initially was to support a custom documentation engine that doesn't accept HTML input.  This is a fairly common situation, though, so **api-documenter** should certainly continue to support Markdown forever, alongside any other output types we may add.

Another idea I've considered is finding an existing documentation generator with very professional templates, and writing an adapter to drive it using .api.json files.  I wouldn't want to bake too much fancy rendering into **api-documenter**, because its goal is to be complete but simple and readable, so people can use its source code as a guide when writing their own custom solution.